cloud_governance/policy/aws/skipped_resources.py

```diff
@@ -109,5 +111,8 @@ def run(self):
         """
         resources_data = self.get_not_delete_resources()
         if self._es_upload.es_host:
-            self._es_upload.es_upload_data(items=resources_data, clear_index_before_delete=True, es_index=self.es_index)
+            start_datetime = datetime.datetime.now().replace(hour=0, minute=0, second=0, microsecond=0)
+            end_datetime = start_datetime - datetime.timedelta(1)
+            self._es_upload.elastic_search_operations.delete_data_in_between_in_es(es_index=self.es_index, start_datetime=start_datetime, end_datetime=end_datetime)
```

Comment:
Add the reason that we delete the data before uploading

Reply:
ok